### PR TITLE
Remove ios-webkit-debug-proxy

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
Since the ios-webkit-debug-proxy only works for iOS 9+ we have decided to show html page in Safari with guide how to debug iOS Cordova applications.

### CODE REVIEW

**Comment by TsvetanMilanov** [comment]
@rosen-vladimirov check the style of the html file.

File: lib/services/debug/darwin-debugger-service.ts [hunk 1]
```diff
@@ -16,35 +16,9 @@ export class DarwinDebuggerService implements IDebuggerService {
 		private $prompter: IPrompter) { }
 
 	public debugIosApplication(applicationId: string): void {
-		// Check if the proxy is installed.
-		try {
-			this.$childProcess.spawnFromEvent("ios_webkit_debug_proxy", ["--help"], "close").wait();
-		} catch (err) {
-			this.$errors.failWithoutHelp("If you want to debug for iOS you need to install ios-webkit-debug-proxy using 'brew install ios-webkit-debug-proxy'.");
-		}
+		let pathToDebuggingGuideHtml = path.join("..", "..", "resources", "debugging", "ios-debug-guide.html");
```
Comment (rosen-vladimirov) [inline lib/services/debug/darwin-debugger-service.ts]: shouldn't this be one level above:
```TypeScript
let pathToDebuggingGuideHtml = path.join(__dirname, "..", "..", "resources", "debugging", "ios-debug-guide.html");
```

**Comment by rosen-vladimirov** [comment]
Linter fails:
16:59:47 >> lib/services/debug/darwin-debugger-service.ts[4, 13]: unused variable: 'shelljs'

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**